src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random RANDOM = new Random();
```

Comment:
According to the checklist, avoid using static methods or fields unless they are constants. Consider changing the 'RANDOM' field to a non-static field.